src/app/containers/RadioSchedule/Canonical/__snapshots__/index.test.jsx.snap

```diff
@@ -375,8 +375,11 @@ exports[`Canonical RadioSchedule renders correctly for a service with a radio sc
   color: #222222;
   -webkit-text-decoration: none;
   text-decoration: none;
-  font-size: 14px;
-  line-height: 18px;
+  font-size: 1.0625rem;
```

Comment:
`LongPrimer` for Arabic script has its own values:

<img width="157" alt="Screenshot 2020-03-24 at 13 11 04" src="https://user-images.githubusercontent.com/17802955/77429095-110eda80-6dd1-11ea-8caf-c6ed85a220de.png">